features/tags_and_wrangling/tag_wrangling_more.feature

```diff
@@ -104,3 +104,105 @@ Feature: Tag wrangling: assigning wranglers, using the filters on the Wranglers
+       And the "Faye Valentine is a sweetie" tag should be canonical
+       And the "Ed is a sweetie" tag should be canonical
+
+  Scenario: Tags that don't exist cause errors
+    Given the following activated tag wrangler exists
+      | login          |
+      | wranglerette   |
+    When I am logged in as "wranglerette"
+    Then visiting "/tags/this_is_an_unknown_tag/edit" it should fail with an error
+      And visiting "/tags/this_is_an_unknown_tag/show" it should fail with an error
+      And visiting "/tags/this_is_an_unknown_tag/feed.atom" it should fail with an error
+
+  Scenario: Banned tags can only be viewed by an admin
+    Given the following typed tags exists
+        | name                                   | type         |
+        | Cowboy Bebop                           | Banned       |
+    When I am logged in as a random user
+     And I view the tag "Cowboy Bebop"
+    Then I should see "Sorry, you don't have permission to access the page you were trying to reach."
+    When I am logged in as an admin
+     And I view the tag "Cowboy Bebop"
+    Then I should not see "Please log in as an admin"
+     And I should see "Cowboy Bebop"
+
+  @javascript
+  Scenario: A user can see hidden tags
+    Given the following typed tags exists
+        | name                                   | type         | canonical |
+        | Cowboy Bebop                           | Fandom       | true      |
+        | Faye Valentine is a sweetie            | Freeform     | false     |
+        | Ed is a sweetie                        | Freeform     | false     |
+      And I am logged in as "first_user" with password "secure_password" with preferences set to hidden warnings and no additional tags
+      And I post the work "Asteroid Blues" with fandom "Cowboy Bebop" with freeform "Ed is a sweetie" with second freeform "Faye Valentine is a sweetie"
+      And I should see "Work was successfully posted."
+      And I am logged in as "second_user" with password "secure_password" with preferences set to hidden warnings and no additional tags
+    When I view the work "Asteroid Blues"
+    When I follow "Show additional tags"
+    Then I should see "Additional Tags: Ed is a sweetie, Faye Valentine is a sweetie"
+     And I should not see "Show additional tags"
+
+  @javascript
+  Scenario: A user can see hidden tags on a series
+
+    Given the following typed tags exists
+        | name                                   | type         | canonical |
+        | Cowboy Bebop                           | Fandom       | true      |
+        | Faye Valentine is a sweetie            | Freeform     | false     |
+        | Ed is a sweetie                        | Freeform     | false     |
+      And I limit myself to the Archive
+      And I am logged in as "first_user" with password "secure_password" with preferences set to hidden warnings and no additional tags
```

Comment:
Since we're not checking the behavior of hidden tags while logged in as `first_user`, this should be okay as plain old `And I am logged in as "first_user"`